rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -124,13 +130,25 @@ public synchronized void processMessage(final Peer sender, @Nonnull final Messag
         message.accept(mv);
 
         long processTime = System.nanoTime() - start;
-        String timeInSeconds = FormatUtils.formatNanosecondsToSeconds(processTime);
+        logProcessingTime(message, processTime);
+    }
 
-        if ((messageType == MessageType.BLOCK_MESSAGE || messageType == MessageType.BODY_RESPONSE_MESSAGE) && BlockUtils.tooMuchProcessTime(processTime)) {
-            loggerMessageProcess.warn("Message[{}] processed after [{}] seconds.", message.getMessageType(), timeInSeconds);
-        } else {
-            loggerMessageProcess.debug("Message[{}] processed after [{}] seconds.", message.getMessageType(), timeInSeconds);
+    private static void logProcessingTime(@Nonnull Message message, long processTime) {
+        String timeInSecondsStr = FormatUtils.formatNanosecondsToSeconds(processTime);
```

Comment:
What do you think about check if the log level is enable or not before do all these operations?